tests/foreman/ui/test_discoveryrule.py

```diff
@@ -254,16 +244,17 @@ def test_positive_list_host_based_on_rule_search_query(
         session.discoveredhosts.apply_action('Auto Provision', [discovered_host['name']])
         assert not session.discoveredhosts.search('name = "{}"'.format(discovered_host['name']))
         values = session.discoveryrule.read_associated_hosts(discovery_rule.name)
+        host_name = values['table'][0]['Name']
         assert values['searchbox'] == f'discovery_rule = "{discovery_rule.name}"'
         assert len(values['table']) == 1
-        assert values['table'][0]['Name'] == provisioned_host_name
-        values = session.host.get_details(provisioned_host_name)
+        assert provisioned_host_name in host_name
+        values = session.host.get_details(host_name)
         assert values['properties']['properties_table']['IP Address'] == ip_address
 
 
 @pytest.mark.tier3
 @pytest.mark.upgrade
-def test_positive_end_to_end(session, module_org, module_location):
+def test_positive_end_to_end(session, module_org, module_location, module_target_sat):
```

Comment:
I think because of the test name this will get collected for stream anyway, but I would still add the E2E marker to be safe / easier to read.

Reply:
I think we shouldn't be considering UI scenarios for the stream marking them e2e. Also, I am not aware of the naming convention that you are talking about here, are all the scenarios with this title getting collected for stream e2e?

Reply:
You're right. Totally forgot that we aren't doing UI tests yet. Nevermind!